pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -26,6 +26,12 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   patches = [
+    # Fix Meson tests that fail when the Nix store is case-sensitive APFS.
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/mesonbuild/meson/pull/11820.patch";
```

Comment:
```suggestion
      url = "https://github.com/mesonbuild/meson/pull/11820.patch";
```

Reply:
Thanks, I fixed the URL and pushed an updated commit.